src/sparse_blas/backends/mkl_common/mkl_handles.hpp

```diff
@@ -61,7 +61,7 @@ struct dense_matrix_handle : public detail::generic_dense_matrix_handle<void*> {
 namespace oneapi::mkl::sparse::detail {
 
 /**
- * Internal sparse_matrix_handle type for MKL backends.
+ * Internal sparse_matrix_handle type for oneMKL backends.
```

Comment:
```suggestion
 * Internal sparse_matrix_handle type for MKLCPU and MKLGPU backends.
```

This makes more sense to me. "oneMKL" alone is still ambiguous as of today.